.eslintrc.json

```diff
@@ -0,0 +1,52 @@
+{
+    "env": {
+      "browser": true,
+      "commonjs": true,
+      "es6": true,
+      "node": true
+    },
+    "parser": "@babel/eslint-parser",
+    "extends": [
+      "next",
+      "next/core-web-vitals",
+      "eslint:recommended",
+      "airbnb",
+      "airbnb/hooks",
+      "plugin:react/recommended",
+      "plugin:import/errors",
+      "plugin:import/warnings",
+      "plugin:jsx-a11y/recommended",
+      "prettier" // Always put prettier at last
+    ],
+    "globals": {
+      "Atomics": "readonly",
+      "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 11,
+      "sourceType": "module",
+      "requireConfigFile": false 
+    },
+    "settings": {
+      "react": {
+        "version": "detect"
+      }
+    },
+    "plugins": ["react", "react-hooks"],
+    "rules": {
+      "react/react-in-jsx-scope": "off",
+      "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+      "react/display-name": 1,
+      "no-param-reassign": ["error", { "props": true, "ignorePropertyModificationsFor": ["state"] }],
+      "import/prefer-default-export": "off",
+      "react/prop-types": "off",
+      "react/function-component-definition" : "off",
+      "jsx-a11y/label-has-associated-control" : "off",
+      "prefer-destructuring" : "off"
+    }
+    
+  }
+  
```

Comment:
EOF - end of file, there should be an empty line in the end of file

Reply:
Ok sir